otelcollector/configmapparser/default-prom-configs/acstorCapacityProvisionerDefaultFile.yml

```diff
@@ -0,0 +1,47 @@
+scrape_configs:
+- job_name: acstor-capacity-provisioner
+  honor_labels: true
+  scrape_interval: 10s

```

Comment:
I see that you're using 10s as the scrape interval instead of the default that we set using $$SCRAPE_INTERVAL$$. Any specific reason for this?

Reply:
No specific reason as such, I just used this as we are using the same in our current setup. Any concerns which you anticipate from deviating from the default?

Reply:
I think using the default setup makes sense to keep everything consistent unless there is a good reason to deviate from it.
The pros for 30 second scrapes are : reduced resource load on the pod and reduced data volume (which can also help with query performance). 

Based on the list of metrics I see in the minimal ingestion profile list I would recommend to use the 30 second value i.e. the `$$SCRAPE_INTERVAL$$` parameter which we replace during config map parsing.

Reply:
@bragi92 Updated the scrape_interval to use default!